.github/workflows/bench.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         - system: rpi4
           name: Arm Cortex-A72 (Raspberry Pi 4) benchmarks
           bench_pmu: PMU
-          archflags: -mcpu=cortex-a72 -DMLK_SYS_AARCH64_SLOW_BARREL_SHIFTER
+          archflags: -mcpu=cortex-a72 -DMLD_SYS_AARCH64_SLOW_BARREL_SHIFTER
```

Comment:
Good catch

Reply:
Well - we don't have that flag yet, but I thought I change it now already.